db/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM postgres:10.3
```

Comment:
Largely copied from the montagu-db configuration, not sure if it will all be relevant. 
I've ended with the database being called modelserver-db (with a dash) and the container being modelserver_db (with an underscore). Can consistify these if you prefer.

Reply:
`modelserver` was just a placeholder name for the prototype. I think makes sense for the container to be called `hint-db` and the db itself just be called `hint` (we need to rename this repo and the gradle project as well, will raise a ticket for that!)